SECURITY_FEATURES.md

```diff
@@ -68,7 +68,7 @@ The root filesystem is marked as read-only and cannot be directly modified by us
 This protects against some container escape vulnerabilities such as [CVE-2019-5736](https://www.openwall.com/lists/oss-security/2019/02/11/2).
 
 The kernel is configured to restart if corruption is detected.
-That allows the system to fail close to when the underlying block device is unexpectedly modified.
+That allows the system to fail closed if the underlying block device is unexpectedly modified.
```

Comment:
I think we need more descriptive text here explaining what this means. I've been peripherally around security for years, and this just read as a typo to me. So a few more words hinting at what this phrase means would be useful.

Reply:
This particular document uses other security-related terms of art - "memory corruption", "container escape", "persistent threat" - without defining them. "Fail closed" feels like a similarly useful term, and comes up in various places around Bottlerocket.

I've tried to elaborate a bit on the value of this design choice; please let me know if that helps to clarify.

Reply:
I believe those other terms are more commonly understood, or at least a little easily understood contextually.

The expanded wording does help a bit, so I do think this is an overall improvement from the original text. Thanks!